_posts/2016-12-02-contributing-to-pulp-blog.md

```diff
@@ -0,0 +1,40 @@
+1) Fork [pulpproject.org respository](https://github.com/pulp/pulpproject.org/) on GitHub.
+
+2) Clone your fork to your computer.
+
+3) Change into the `pulpproject.org` directory and checkout `gh-pages` branch
+
+4) Create a new file in the `_posts` directory. The name for the file should be in the following
+   format: `Year-Month-Day-short-name-for-post.md`
+
+5) Paste the following template into your new blog file.
```

Comment:
I think it's more important to identify what frontmatter is and give the example rather than a copy/paste example.